src/libs/kata-types/src/config/hypervisor/ch.rs

```diff
@@ -103,12 +102,6 @@ impl ConfigPlugin for CloudHypervisorConfig {
                 return Err(eother!("Valid CH jailer path list should be empty"));
             }
 
-            if !ch.blockdev_info.disable_block_device_use
-                && ch.blockdev_info.block_device_driver == VIRTIO_BLK_MMIO
-            {
-                return Err(eother!("CH doesn't support virtio-blk-mmio"));
```

Comment:
@jodh-intel was this support recently added in CH, or was there a reason it was not previously supported in runtime-rs?

Reply:
@cmaf - Adding that check originally was an oversight, hence I zapped it.

Reply:
Could this go in a separate commit providing an explanation for it in the commit message?

Reply:
Done.